Stellar-ledger-entries.x

```diff
@@ -515,10 +515,10 @@ struct ContractCodeEntry {
     opaque code<>;
 };
 
-struct ExpirationEntry {
-    // Hash of the LedgerKey that is associated with this ExpirationEntry
+struct TTLEntry {
+    // Hash of the LedgerKey that is associated with this TTLEntry
     Hash keyHash;
-    uint32 expirationLedgerSeq;
+    uint32 liveUntilLedgerSeq;
```

Comment:
Is this field inclusive or exclusive?

Reply:
Inclusive, such that an entry e is live iff `e.liveUntilLedgerSeq >= currentLedgerSeq`

Reply:
Then I think the name is misleading. I think it's reasonable to interpret until to mean exclusive. It's alive until the given ledger seq where it will now no longer be live.

Reply:
> Then I think the name is misleading. I think it's reasonable to interpret until to mean exclusive. It's alive until the given ledger seq where it will now no longer be live.

Until means up to that ledger, right? We can also consider `liveTilLedgerSeq`.

Reply:
I think the issue with this name is that we have to name is such that "this entry lives until X." Previously we used the language "expires on". With the new naming convention, we can't use the "expires on" phrasing because there is not a unified verb for what happens when a TEMPORARY and PERSISTENT entry "expire." When they are no longer live, TEMPORARY entries get deleted, while PERSISTENT entries get archived. We could use something like "deathLedger", but I'd like to avoid terminology suggesting entries are deleted. Given that we no longer have a unifying verb (i.e. expired), I think using the live until phrasing is appropriate. This is mostly an implementation detail, I expect most devs to interface with this information via and entries TTL that is derived from this field. I don't thing `liveTilLedgerSeq` helps solve Leigh's issue since it's the same word. Do you have any suggestions that indicate this field to be inclusive @leighmcculloch?